groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/lanthanides/SamariumChain.groovy

```diff
@@ -0,0 +1,28 @@
+import globals.Globals
+
+import static gregtech.api.unification.material.Materials.*;
+import gregtech.api.unification.material.MarkerMaterials;
+import static gregtech.api.unification.ore.OrePrefix.dye;
+
+BR = recipemap('batch_reactor')
+REACTION_FURNACE = recipemap('reaction_furnace')
+
+BR.recipeBuilder()
+        .fluidInputs(fluid('samarium_amalgam') * 1000)
+        .fluidInputs(fluid('hydrochloric_acid') * 5000)
+        .outputs(metaitem('dustSamariumChloride') * 4)
```

Comment:
Wikipedia backs up that this is soluble enough to be contained in a solution (92.4g/100mL), although I think mercury chloride wouldn't be included due to the common ion effect.